climpred/tests/conftest.py

```diff
@@ -5,7 +5,12 @@
 from climpred import PerfectModelEnsemble
 from climpred.tutorial import load_dataset
 
-# ordering: PM MPI, CESM; xr.Dataset, xr.DataArray; 1D, 3D; generic xr.objects
+
+def set_cftime_to_int_dim(ds, dim, freq='YS'):
```

Comment:
this only works for consecutive inits. do you mind changing to? Note that I currently need `NoLeap` or ``Leap` to get consistent `dt.dayofyear` (my PR soon)
```
def set_cftime_to_int_dim(ds, dim, calendar='DatetimeNoLeap'):
    import cftime
    ds[dim] = [getattr(cftime, calendar)(i,1,1) for i in ds[dim].values]
    return ds